web/source/osk/oskManager.ts

```diff
@@ -357,123 +353,14 @@ namespace com.keyman.osk {
 
       // Add footer element to OSK only for desktop browsers
       if(util.device.formFactor == 'desktop') {
-        this._Box.appendChild(this.resizeBar());
+        this.desktopResizeBar = new layouts.TargetedResizeBar();
+        this._Box.appendChild(this.desktopResizeBar.element);
         // For other devices, adjust the object heights, allowing for viewport scaling
       } else {
         this.vkbd.adjustHeights(this);
       }
     }
 
-    /**
-     * Create a control bar with title and buttons for the desktop OSK
-     */
-    controlBar(): HTMLDivElement {
-      // TODO: merge with _TitleBarInterior?
```

Comment:
Challenge accepted.  There's only one version (a blend of the two) in the final result.